example/src/reducers/componentState/ScrollableState.ts

```diff
@@ -0,0 +1,37 @@
+import { IDocument } from '../../Types';
```

Comment:
Why is this file here?

Reply:
Yes, you are right, I am removing that file from example folder now.